katsdpcontroller/product_config.py

```diff
@@ -704,8 +723,10 @@ class NgcBaselineCorrelationProductsStream(BaselineCorrelationProductsStreamBase
     stream_type: ClassVar[str] = 'ngc.baseline_correlation_products'
     _valid_src_types: ClassVar[_ValidTypes] = {'ngc.antenna_channelised_voltage'}
 
-    def __init__(self, name: str, src_streams: Sequence[Stream], *,
-                 int_time: float) -> None:
+    def __init__(self, name: str,
+                 src_streams: Sequence[Stream], *,
+                 int_time: float,
```

Comment:
`int_time` was confusing to me for a bit. Why is it called `int` time if it's a `float`? I take it you mean integration time?

Reply:
:rofl: That's the official CBF term. 😛

Reply:
Yes, integration time. IIRC that spelling is part of the CBF-CAM ICD.

Reply:
I guess it would help if I read the document! 😊